packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -361,12 +361,37 @@ type Shortcut = {
   onClick?: (ctx: Omit<SetupContext, 'expose'>) => void
 }
 
+const arrangeDatesShortcuts = (selectedValue) => {
+  const isAlreadyArray = Array.isArray(props.parsedValue as Dayjs[])
+  if (!isAlreadyArray) {
+    return emit([selectedValue], true)
```

Comment:
the second params of `emit` used to keep popper visible when `multiple` is true

Reply:
The second parameter seems to be omitted, here is the judgment logic
https://github.com/element-plus/element-plus/blob/b0709e1d83758b1ea13b57f7cc26b2c108cc8755/packages/components/date-picker/src/date-picker-com/panel-date-pick.vue#L320-L321

Reply:
there are not same. this line is `handleDatePick`, and the line of 367 is `handleShortcutClick`.
![image](https://user-images.githubusercontent.com/30046649/231082565-7fbace86-05e0-4262-95f0-1f37c126ff82.png)